src/main/java/sirius/biz/jobs/batch/file/XMLImportJob.java

```diff
@@ -31,9 +30,9 @@
  */
 public abstract class XMLImportJob extends FileImportJob {
 
-    public static final Parameter<String> XSD_SCHEMA_PARAMETER =
+    protected static final SelectStringParameter XSD_SCHEMA_PARAMETER_BUILDER =
```

Comment:
provide a factory method and or the name as shared constant.

maybe also prevent a lookup in the string list for when reading the parameter as the verification is done before and the value can be assumed to be correct. in other words a SelectStringParameter should behave like a StringParameter when it comes to reading the value from the job...

Reply:
Builder delivered by static method.